### PR TITLE
doc: add send command documentation

### PR DESCRIPTION
**Pull request by ftreguer**
## Proposed changes

Add send command documentation [#1477](https://spacelephant.tpondemand.com/entity/1477-documenter-la-commande-cli-greater-send)

### CODE REVIEW

File: docs/uns-use-the-network/cli.md [hunk 1]
```diff
@@ -635,3 +635,44 @@ $ uns did-resolve --network devnet @unik:individual/bob?postal_address -f raw
...
 42 quai Malakoff, 44000 Nantes
 ```
+
+### `send`
+
+#### Introduction
+Send owned UNS protocol tokens to another wallet.
+
+#### Arguments
+- `AMOUNT` (required):  The quantity of tokens to send to the recipient.
```
Comment (Nigui) [inline docs/uns-use-the-network/cli.md]: we must specify that it's in UNS (not in a smaller unit of value)

File: docs/uns-use-the-network/cli.md [hunk 2]
```diff
@@ -635,3 +635,44 @@ $ uns did-resolve --network devnet @unik:individual/bob?postal_address -f raw
...
+#### Parameters
+
+- `--to` (required): The recipient address.
+- `--no-check`: Allow sending tokens to an address that do not exists on chain yet.
+- `--fees-included`: Specify that the fees must be deducted from the amount. By default the fees are paid on top.
+- `--await`: Number of blocks to wait to get confirmed for the success. Default to 3. 0 for immediate return.
+- `--confirmations`: Number of confirmations to wait to get confirmed for the success. Default to 1.
+- `--passphrase`: The passphrase of the owner of unik. If you do not enter a passphrase you will be prompted for it.
+- `--secondPassphrase`: The second wallet passphrase. If you have created a second passphrase on your wallet, you have to enter it.
+- `--fee`: Specify a dynamic fee in UNSat. Defaults to 100000000 UNSat = 1 UNS.
```
Comment (Nigui) [inline docs/uns-use-the-network/cli.md]: satoUNS, UNSat, ... we must choose a be consistent

Comment (ftreguer) [reply docs/uns-use-the-network/cli.md]: @laurentlourenco

File: docs/uns-use-the-network/cli.md (hunk 1 above)
Comment (ftreguer) [reply docs/uns-use-the-network/cli.md]: @laurentlourenco

**Pull request merged**